### PR TITLE
Sbt plugin prints errs if exit code is not 0

### PR DESCRIPTION
**Pull request by kornilova-l**
Sbt plugin used to throw an exception if bindgen exit code is not 0.
But exception does not contain useful information.

This will be helpful for debugging

### CODE REVIEW

File: tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala [hunk 1]
```diff
@@ -129,13 +131,28 @@ object Bindgen {
...
+      )
+      val process   = Process(cmd).run(io)
+      val exitValue = process.exitValue()
+
+      if (exitValue != 0) {
+        System.err.println(
+          s"Bindgen finished with exit code $exitValue.\n$errs")
+      }
+
+      new Bindings(output, errs)
```
Comment (jonas) [inline tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala]: I've created #144 that I think is a bit cleaner. It also adds messages for the `require` calls which was a problem I had when developing the bindings and sbt plugin.

File: tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala [hunk 2]
```diff
@@ -65,7 +66,8 @@ object Bindgen {
                                 packageName: Option[String] = None,
                                 excludePrefix: Option[String] = None,
                                 extraArg: Seq[String] = Seq.empty,
-                                extraArgBefore: Seq[String] = Seq.empty)
+                                extraArgBefore: Seq[String] = Seq.empty,
+                                config: Option[File] = None)
```
Comment (jonas) [inline tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala]: This change looks unrelated since there is no setter.

Comment (kornilova-l) [reply tools/src/main/scala/org/scalanative/bindgen/Bindgen.scala]: Oops, it's from different commit

**Comment by kornilova-l** [comment]
The PR is closed because #144 solves the problem better
